api/src/main/scala/dcos/metronome/api/ApiModule.scala

```diff
@@ -24,7 +25,10 @@ class ApiModule(
   pluginManager:      PluginManager,
   httpErrorHandler:   HttpErrorHandler,
   assets:             Assets,
-  launchQueueService: LaunchQueueService) {
+  launchQueueService: LaunchQueueService,
+  actorsModule:       ActorsModule) {
```

Comment:
I wasn't sure if we should use the same Actor system.

Reply:
What's wrong with using the same ActorSystem?

Reply:
yeah, I think this is fine